cl/lib/solr_core_admin.py

```diff
@@ -57,7 +57,13 @@ def create_default_cores():
                             'audio_schema.xml'),
         instance_dir='/usr/local/solr/example/solr/audio',
     )
-
+    create_solr_core(
+        core_name='dockets',
+        data_dir=os.path.join(settings.INSTALL_ROOT, 'Solr', 'data_dockets'),
+        schema=os.path.join(settings.INSTALL_ROOT, 'Solr', 'conf',
+                            'dockets_schema.xml'),
+        instance_dir='/usr/local/solr/example/solr/dockets',
+    )
 
```

Comment:
You should have one more line here. (Total nitpick, I know, but it'll prevent me from doing a one-line fix someday down the road.)